docs/schema.md

```diff
@@ -118,11 +118,11 @@ Including a `project` entry marks the service as a *project*:
 - It will build and run locally using the .NET project during development. 
 - It will be packaged and deployed during deployments.
 
-#### `dockerImage` (string)
```

Comment:
The docs should call out that this might still be `dockerImage` depending on your build. For instance our instructions point at a version number that will use `dockerImage`.

Reply:
Yes this is my fear with this change. What do we want to do here?